dotCMS/src/main/java/com/dotmarketing/util/MaintenanceUtil.java

```diff
@@ -689,9 +689,14 @@ public static void fixImagesTable() throws SQLException{
 	 *             An error occurred when executing the fix queries.
 	 */
 	public static void deleteOrphanContentTypeFields() throws SQLException {
+	    String query = "DELETE FROM field WHERE NOT EXISTS (SELECT * FROM structure WHERE structure.inode = field.structure_inode)";
 		DotConnect dc = new DotConnect();
-		dc.executeStatement("DELETE FROM field WHERE structure_inode NOT IN (SELECT inode FROM structure)");
-		dc.executeStatement("DELETE FROM field WHERE structure_inode NOT IN (SELECT inode FROM inode)");
+		dc.setSQL(query);
+		dc.getResult();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getResult"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)